### PR TITLE
[Localization] Translate For all the files in `zh-CN\get-involved`

### PR DESCRIPTION
**Pull request by ghost**
All the files there are totally completed into Chinese.

### CODE REVIEW

**Comment by fhemberger** [comment]
@nodejs/nodejs-cn Could you please review this PR so we can merge it? Thank you!

File: locale/zh-cn/get-involved/collab-summit.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: 协作者峰会
+layout: contribute.hbs
+---
+
+# 协作者峰会
```
Comment (willin) [inline locale/zh-cn/get-involved/collab-summit.md:6]: \n

File: locale/zh-cn/get-involved/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: 加入我们
+layout: contribute.hbs
+---
+
+# 我们鼓励来自社区的各种贡献
+
+Node.js 是个包容的大家庭，因此我们鼓励用户各施所长。 若你想 [汇报问题](https://github.com/nodejs/node/issues) 或者 [协助编辑文件及贡献代码](/zh-tw/get-involved/contribute/)，那就来对地方了，看看下列的社区资源是不是能帮到你：
```
Comment (willin) [inline locale/zh-cn/get-involved/index.md]: `zh-tw` in link

**Review by willin — APPROVED** [review]
LGTM

File: locale/zh-cn/get-involved/code-and-learn.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: 代码 + 学习
+layout: contribute.hbs
+---
+
+# 代码 + 学习
```
Comment (XadillaX) [inline locale/zh-cn/get-involved/code-and-learn.md]: I think 'Code' is a verb. `编码` may be better.

File: locale/zh-cn/get-involved/code-and-learn.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+“代码 & 学习”活动将让你随着 Node.js 核心贡献者们共同起步（或飞得更高）。无论你是你第一次还是第二、第三或是第四次为 Node.js 核心做出贡献，经验老道的贡献者们都能提供协助，他们也能现场即兴带领大家攻略 Node.js 核心源码。
+
+## 即将到来的代码 + 学习活动
+
+- 待定
+
+## 已经结束的代码 + 学习活动
+
+- [Oakland on April 22, 2017](https://medium.com/the-node-js-collection/code-learn-learn-how-to-contribute-to-node-js-core-8a2dbdf9be45)
```
Comment (XadillaX) [inline locale/zh-cn/get-involved/code-and-learn.md:16]: 2017-04-22，美国奥克兰

File: locale/zh-cn/get-involved/code-and-learn.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+- Shanghai at JSConf.CN: July 2017
```
Comment (XadillaX) [inline locale/zh-cn/get-involved/code-and-learn.md:17]: `2017-07，上海 JSConfCN` may be better?

File: locale/zh-cn/get-involved/index.md (hunk 2 above)
Comment (ghost) [reply locale/zh-cn/get-involved/index.md]: Fixed

File: locale/zh-cn/get-involved/collab-summit.md (hunk 1 above)
Comment (ghost) [reply locale/zh-cn/get-involved/collab-summit.md:6]: Fixed

File: locale/zh-cn/get-involved/code-and-learn.md (hunk 4 above)
Comment (ghost) [reply locale/zh-cn/get-involved/code-and-learn.md:16]: I don't think it needs translating these addresses, locations……ect. Please refer 'zh-TW', 'uk' 's translations.

File: locale/zh-cn/get-involved/code-and-learn.md (hunk 5 above)
Comment (ghost) [reply locale/zh-cn/get-involved/code-and-learn.md:17]: I don't think it needs translating these addresses, locations……ect. Please refer 'zh-TW', 'uk' 's translations.

**Pull request merged**